perl-algorithm-diff.yaml

```diff
@@ -0,0 +1,46 @@
+# Generated from https://git.alpinelinux.org/aports/plain/community/perl-algorithm-diff/APKBUILD
+package:
+  name: perl-algorithm-diff
+  version: "1.201"
+  epoch: 0
+  description: Compute 'intelligent' differences between two files / lists
+  copyright:
+    - license: GPL-1.0-or-later OR Artistic-1.0-Perl
+  dependencies:
+    runtime:
+      - perl
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - perl
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 0022da5982645d9ef0207f3eb9ef63e70e9713ed2340ed7b3850779b0d842a7d
+      uri: https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Algorithm-Diff-${{package.version}}.tar.gz
+
+  - uses: perl/make
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: perl/cleanup
+
+  - uses: strip
+
+subpackages:
+  - name: perl-algorithm-diff-doc
+    pipeline:
+      - uses: split/manpages
+    description: perl-algorithm-diff manpages
+
+update:
+  enabled: false # latest releases mismatches in release monitor and github release is not maintained to latest version
```

Comment:
I wonder if this means the project is not maintained anymore? If not should we be adding it to wolfi?

Reply:
I think I discussed this with @kaniini and we moved to enabling it false but it might be not maintained because is completed? It is needed for `perl-text-diff` package which is in turn along with others is needed for accp package so not sure would should be the next step here